packages/wallets/src/evm/connectors/smart-wallet/index.ts

```diff
@@ -35,6 +35,7 @@ export class SmartWalletConnector extends Connector<SmartWalletConnectionArgs> {
   private accountApi: AccountAPI | undefined;
   personalWallet?: EVMWallet;
   chainId?: number;
+  approving?: boolean;
```

Comment:
whats this for?

Reply:
making sure not to go through approval flow if a tx is sent as approval is happening
on second thought probably should block (wait) all of sendTransaction while approving is true
is there a WaitUntil -> type thing in ts
then again we dont wait until something is deployed in ts right? if i send 2 in a row it'll add initcode twice potentially?

Reply:
but are you using this variable? i see there's one in the signer class already, this one doesnt seem used?